src/styles.variables.scss

```diff
@@ -0,0 +1,6 @@
+@import "~@angular/material/theming";
+
+$amber: mat-color(mat-palette($mat-amber));
+$warn-background: #ffe3e6;
+$login-background: #f6f2fc;
```

Comment:
Ideally, we don't have to relate the CSS variable to the particular component (e.g. login) as we could use it somewhere else. It's better to call it primary-background for example (up to you, if you would like to use this type of background as common)